server/bin/pbench-backup-tarballs.py

```diff
@@ -240,18 +242,26 @@ def backup_to_s3(s3_obj, logger, controller_path, controller, tb, tar, resultnam
     except Exception:
         s3_md5 = None
```

Comment:
If we get an Exception here, is that normal?  We might want to consider logging something.

Reply:
Yes and no: this is actually fixed by the last commit in the series. If the key is not found, that should be normal (but it was not, hence the commit). However, there may be other exceptions that are not normal.